manifests/plugin.pp

```diff
@@ -1,6 +1,7 @@
 # Installs a smart proxy plugin package
 define foreman_proxy::plugin(
-  $package = "${foreman_proxy::plugin_prefix}${title}",
+  $version = 'installed',
+  $package = "${foreman_proxy::params::plugin_prefix}${title}",
```

Comment:
Sorry about that, I didn't noticed. I've made the changes on my side a few days ago and I did it, it seems, on an older code, without doing a sync first. Fixed now